packages/app/src/app/pages/NewDashboard/Components/Header/index.tsx

```diff
@@ -52,7 +54,7 @@ export const Header = ({
           {title}
         </Text>
       ) : (
-        <Breadcrumbs activeTeam={activeTeam} path={path} />
+        <Breadcrumbs repos={repos} param={path} />
```

Comment:
`activeTeam` needs to be passed in the crumbs, because it's used to determine the URL (`teamId` is passed in the URL).

Reply:
done